modules/fedimint-ln-common/src/api.rs

```diff
@@ -44,7 +44,7 @@ where
     }
 
     async fn fetch_contract(&self, contract: ContractId) -> FederationResult<ContractAccount> {
-        self.request_current_consensus("wait_account".to_string(), ApiRequestErased::new(contract))
+        self.request_current_consensus("account".to_string(), ApiRequestErased::new(contract))
```

Comment:
I'm not sure about this change.

Reply:
Also, the difference between a "contract" and a "contract account" has always been confusing. Not much to do here, but just making note.

Reply:
This actually seems like a good change. `fetch_contract` should probably just get the contract and fail if it's not there. I'm not aware of any case where the gateway will go looking for a contract before it has been created and need to wait for it to show up.

Reply:
The gateway might not, but does the lightning client? This is shared between both. I don't think this is necessary, I would probably prefer to revert unless others have strong opinions.